sensor_msgs/LaserScan laser_scan
bool[] reflector_status
bool[] reflector_median
bool[] intrusion

